src/components/RecipientInfo.vue

```diff
@@ -0,0 +1,156 @@
+<template>
+	<div class="recipient-info">
+		<!-- For a single recipient -->
+		<div v-if="recipients.length === 1" class="recipient-single">
+			<div class="recipient-header">
+				<div class="recipient-avatar">
+					<Avatar :display-name="recipients[0].label"
+						:email="recipients[0].email"
+						:size="55"
+						:disable-tooltip="true"
+						:disable-menu="true"
+						:url="recipients[0].photoUrl" />
+				</div>
+				<div class="recipient-details">
+					<DisplayContactDetails :email="recipients[0].email" />
+				</div>
+			</div>
+		</div>
+
+		<!-- For multiple recipients -->
+		<div v-else class="recipient-multiple">
+			<div v-for="(recipient, index) in recipients" :key="index" class="recipient-item">
+				<div class="recipient-header">
+					<div class="recipient-avatar">
+						<Avatar :display-name="recipient.label"
+							:email="recipient.email"
+							:size="55"
+							:disable-tooltip="true"
+							:disable-menu="true" />
+					</div>
+					<div v-if="!expandedRecipients[index]" class="recipient-name">
```

Comment:
Why is this section not present for a single recipient?

Reply:
you mean the v-if? That is because when its one recipient only, it shouldnt be collapsed. It collapses when a second one is added.